docs/docs/getting-started/deployment/configuration.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Configuration
+sidebar_label: Configuration
+---
+
+## Airy Core Configuration File
+
+The `infrastructure/airy.tpl.yaml` file contains examples of all possible configuration options. Copy and rename it to `infrastructure/airy.yaml` and modify it according to your environment:
+
+```bash
+cd infrastructure
+cp airy.tpl.yaml airy.yaml
+```
+
+We will guide you through the different sections so you can make the changes you are looking for. Keys can also be omited for services which you do not wish to configure.
+
+### Global
+
+- **appImageTag**
+
+  image tag of the container images for the `Airy Components`
+
+- **containerRegistry**
+
+  The images of the `Airy Components` are stored in the [Github Container Registry](https://docs.github.com/en/packages/guides/about-github-container-registry).
```

Comment:
The subject here should be something like "The URL of the container registry" I think (unless I'm missing something)

Reply:
I just don't want people to encourage changing it but yeah it's their fault if they do so I'll change it